luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -196,17 +213,28 @@ class SeqrSVGenotypesSchema(SeqrGenotypesSchema):
 
     def _genotype_fields(self):
         is_called = hl.is_defined(self.mt.GT)
+        was_previously_called = hl.is_defined(self.mt.CONC_ST) & ~self.mt.CONC_ST.contains("EMPTY")
+        num_alt = self._num_alt(is_called)
+        prev_num_alt = PREVIOUS_GENOTYPE_N_ALT_ALLELES.get(hl.set(self.mt.CONC_ST), -1)
+        discordant_genotype = (num_alt != prev_num_alt) & (prev_num_alt > 0)
+        novel_genotype = (num_alt != prev_num_alt) & (prev_num_alt == 0)
         return {
             'sample_id': self.mt.s,
             'gq': self.mt.GQ,
             'cn': self.mt.RD_CN,
-            'num_alt': self._num_alt(is_called)
+            'num_alt': num_alt,
+            'prev_num_alt': hl.or_missing(discordant_genotype, prev_num_alt),
+            'new_call': hl.or_missing(is_called, ~was_previously_called | novel_genotype | discordant_genotype),
```

Comment:
  | new_call | new_genotype
-- | -- | --
Absent in previous callset. | T | F
Concordant Genotype | F | F
Novel Genotype | T | F
Absent Genotype | F | T
Discordant Genotype | F | T


Should I have instead left this `new_call` field as is and just modified the filter function in `samples_new_call` to filter for samples that had either `new_call` OR `prev_num_alt != missing`?  That made less sense to me than just expanding the definition here.

Reply:
yeah, you should have left `new_call` as it was and modified `samples_new_call`. `samples_new_call` is what seqr will use to filter data at search time, the genotype field is what it will use to display data. For filtering, the ticket specifies that the existing filtering for "new call" should return new genotypes as well, so we want to change the `samples_new_call` field to include all the samples that search should match on. But then for display we will display new call and new genotype differently from each other

Reply:
👍 Yep that makes sense.